spec/controllers/invitations_controller_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'spec_helper'
+
+describe InvitationsController do
+
+  let(:user) { Fabricate(:user) }
+
+  context 'for signed-in user' do
+
+    before do
+      login_user user
+    end
+
+    describe 'GET :new' do
+      it 'sets a new invitation linked to the current user' do
+        get :new
+        expect(assigns(:invitation)).to be_instance_of(Invitation)
+        expect(assigns(:invitation).user).to eq user
+      end
+    end
+
+    describe 'POST :create' do
+
+      before do
+        ActionMailer::Base.deliveries.clear
+      end
+
+      context 'for new persons' do
+
+        before do
+          post :create, invitation: Fabricate.attributes_for(:invitation, email: 'new-user@domain.com', message: 'Have fun!')
+        end
+
+        it 'redirects to the new template' do
```

Comment:
You don't redirect to a template, you redirect to a path. Templates are rendered, and not necessarily by the action of the same name.